packages/sign-client/test/sdk/client.spec.ts

```diff
@@ -493,6 +493,91 @@ describe("Sign Client Integration", () => {
           await throttle(1_000);
           await deleteClients(clients);
         });
+
+        it("can set tvf params", async () => {
```

Comment:
Is that the only test case we need ? What about checking validation errors etc ?

Reply:
added extra validations for the different namespaces and result structures. Validation errors are tested with the other tests where different methods/values are used and don't assign tvf